README.md

```diff
@@ -1 +1,15 @@
-# design-pattern
\ No newline at end of file
+# design-pattern
+
+designing patterns are as follows 
+
+1) creational:
+     i- Factory
+     ii-Abstract Factory
+     ii-Builder
+2) Structural
+      i- Decorator
```

Comment:
How is selection of paymentMode the function of a decorator? 
Why do you have a paypal and tez decorator? What is the function of these decorators?
Incorrect pattern.